index.html

```diff
@@ -62,9 +62,11 @@
     <div class="primary-nav" itemscope itemtype="http://schema.org/Organization">
       <a class="logo-btn" href="/" itemprop="url">
         <picture>
-            <source srcset="app/assets/images/logo-macile.webp" type="image/webp">
-            <img class="logo-img" src="app/assets/images/logo-macile.jpg" itemprop="logo">
-          </picture>
+        	<source srcset="app/assets/images/logo-macile-116x125.webp" type="image/webp">
+        	<source srcset="app/assets/images/logo-macile-167x180.webp" type="image/webp">
+          <source srcset="app/assets/images/logo-macile-84x90.webp" type="image/webp">
+          <img class="logo-img" src="app/assets/images/logo-macile.jpg" itemprop="logo">
+        </picture>
```

Comment:
There are two different ways to handle responsive images, using `picture` and using `img srcset=""`. Each one has pros and cons that we should be familiar with. 

I recommended reviewing literature on responsive images, it will be very valuble for the project in the long run. 

here's an article on csstricks about srcset https://css-tricks.com/responsive-images-youre-just-changing-resolutions-use-srcset/

Reply:
ps. let's use absolute urls. will save us a lot of pain in the long run. 
pps. install a simple http server `npm install -g http-server` 

in the projoject directory, you can run `http-server` and it will give you a local server. this is good for testing and you should run it and check your work locally before requesting a review on a PR.